.github/workflows/build-and-push-ui-images.yml

```
@@ -0,0 +1,79 @@
+  IMG_ORG: kubeflow
+  IMG_UI_REPO: model-registry/ui # this image is intended for local development, not production
+  DOCKER_USER: ${{ github.actor }}
+  DOCKER_PWD: ${{ secrets.GITHUB_TOKEN }}
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)